library/ssl_tls13_generic.c

```diff
@@ -1328,6 +1328,39 @@ int mbedtls_ssl_tls13_write_change_cipher_spec( mbedtls_ssl_context *ssl )
 
 #endif /* MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE */
 
+/* Early Data Extension
+ *
+ * struct {
+ *   select ( Handshake.msg_type ) {
+ *     case new_session_ticket:   uint32 max_early_data_size;
+ *     case client_hello:         Empty;
+ *     case encrypted_extensions: Empty;
```

Comment:
```suggestion
 *   select ( Handshake.msg_type ) {
 *     ...
 *     case client_hello:         Empty;
 *     case encrypted_extensions: Empty;
```

Reply:
To keep aligned with the spec RFC 8466, I prefer to keep it here even we don't parse it for the time being.

Reply:
`max_early_data_size` is not covered by this function.  It is common comments style.

ssl_msg.c line 2421, ssl_tls13_client.c 1235,  all non relative parts are replace with `...`